src/main/java/org/jcodec/common/model/Picture.java

```diff
@@ -58,14 +68,14 @@ public Picture(int width, int height, byte[][] data, ColorSpace color, Rect crop
     }
 
     public static Picture copyPicture(Picture other) {
-        return new Picture(other.width, other.height, other.data, other.color, other.crop);
+        return new Picture(other.width, other.height, other.data, other.lowBits, other.color, 0, other.crop);
     }
     
     public static Picture create(int width, int height, ColorSpace colorSpace) {
-        return createCropped(width, height, colorSpace, null);
+        return createCropped(width, height, 0, colorSpace, null);
     }
 
-    public static Picture createCropped(int width, int height, ColorSpace colorSpace, Rect crop) {
+    public static Picture createCropped(int width, int height, int lowBitsNum, ColorSpace colorSpace, Rect crop) {
```

Comment:
we can only change public interfaces `Picture.createCropped` if we are planning to release an api-incompatible version
therefore i suggest we keep `public static Picture createCropped(int width, int height, ColorSpace colorSpace, Rect crop)` (i.e. method without `lowBitsNum`)
and add additional `public static Picture createCroppedHiBD(int width, int height, int lowBitsNum, ColorSpace colorSpace, Rect crop)`